specifications/android.md

```diff
@@ -49,18 +55,21 @@ The exposed broadcast receiver of the distributor application MUST handle 2 diff
 
 ### org.unifiedpush.android.distributor.REGISTER
 
-The connector send this action to register to push messages. The intent MUST contain 2 String extras:
-* application: with the end user application package name. The distributor MUST be able to handle many connections with a single application.
-* token: with a random token to identify the connection between the connector and the distributor. It MUST be unique on distributor side.
+The connector sends this action to register to push messages. The intent MUST contain 2 extras:
+* application (String): with the end user application package name. The distributor MUST be able to handle many connections with a single application.
```

Comment:
The opposite sounds weird to me. Maybe because it is not my mother tongue

Reply:
I removed the withs, it matches the rest of the spec. I also replaced `random` by `randomly generated` because `random` could be interpreted as `any` I believe.